HtmlCssSample.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="HtmlCssSample.css">
+        <title>Sample Form</title>
+    </head>
+    <body>
+        <section class="FormBody">
+            <form>
+                <label for "name">Name</label>
+                
+                <input type="text" id="name" placeholder="Name" class="InputDetails">
+                
+                <label for "userName">User Name</label>
+                
+                <input type="text" id="userName" placeholder="User Name" class="InputDetails">
+                
+                <label for "passWord">Password</label>
+                
+                <input type="password" id="passWord" placeholder="Password" class="InputDetails">
+
+                <label for "passWordHint">Password Hint</label>
+                <textarea placeholder="password hint" id="passWordHint" class="InputDetails"></textarea>
```

Comment:
The textarea element should have a rows and cols attribute to specify its size.

Reply:
Resolved